libs/core/src/lib/resizable-card-layout/resizable-card-layout.component.ts

```diff
@@ -0,0 +1,497 @@
+    /** Drag of card can be disabled. Default is true */
+    @Input()
+    draggable = true;
+
+    /** Object for setting each card property */
+    @Input()
+    layoutConfig: ResizableCardLayoutConfig;
+
+    /** Available layout width */
+    @Input()
+    get layoutSize(): LayoutSize {
+        return this._layout;
+    }
+
+    set layoutSize(layoutSize: LayoutSize) {
+        this._layout = layoutSize;
+        this._setLayoutColumns(layoutSize);
+    }
+
+    /** Emits when card resize is reached to one new step in horizontal or vertical direction */
+    @Output()
+    stepChange: EventEmitter<ResizedEvent> = new EventEmitter<ResizedEvent>();
+
+    /** Emits when card is dragged and dropped at different position in layout */
+    @Output()
+    dropped: EventEmitter<CardDropped> = new EventEmitter<CardDropped>();
+
+    /** Emits when card is still resizing */
+    @Output()
+    resizing: EventEmitter<ResizingEvent> = new EventEmitter<ResizingEvent>();
+
+    /** Emits when card resize is completed */
+    @Output()
+    resized: EventEmitter<ResizedEvent> = new EventEmitter<ResizedEvent>();
+
+    /** Emits when card height is reduced to show only header */
+    @Output()
+    miniHeaderReached: EventEmitter<ResizedEvent> = new EventEmitter<ResizedEvent>();
+
+    /** Emits when minimum height of card content area is reached */
+    @Output()
+    miniContentReached: EventEmitter<ResizedEvent> = new EventEmitter<ResizedEvent>();
+
+    /** Emits when layout changes */
+    @Output()
+    layoutChange: EventEmitter<ResizableCardLayoutConfig> = new EventEmitter<ResizableCardLayoutConfig>();
+
+    /** @hidden */
+    // @ViewChildren(ResizableCardItemComponent)
+    // resizeCardItems: QueryList<ResizableCardItemComponent>;
+
+    /** @hidden */
+    @ContentChildren(ResizableCardItemComponent)
+    resizeCardItems: QueryList<ResizableCardItemComponent>;
+
+    /** @hidden */
+    layoutWidth: number;
+
+    /** @hidden */
+    layoutHeight: number;
+
+    dragStartDelay = DRAG_START_DELAY;
+
+    /** @hidden Number of columns in layout. considering 1 column width 20rem */
+    private _columns: number;
+
+    /** @hidden Layout padding. will be added to first card padding in every row */
+    private _paddingLeft: number;
+
+    /** @hidden Stores height of each column on card arrangement */
+    private _columnsHeight: Array<number>;
+
+    /** @hidden Available screen layout */
+    private _layout: LayoutSize;
+
+    /** @hidden Stores sorted card before placing in layout */
+    private _sortedCards: Array<ResizableCardItemComponent>;
+
+    /** @hidden FocusKeyManager instance */
+    private _keyboardEventsManager: FocusKeyManager<ResizableCardItemComponent>;
+
+    /** @hidden  */
+    private _destroy$ = new Subject<boolean>();
+
+    constructor(private readonly _changeDetectorRef: ChangeDetectorRef, private readonly _elementRef: ElementRef) {}
```

Comment:
`_changeDetectorRef: ChangeDetectorRef` should be `_cd: ChangeDetectorRef` for consistency

Reply:
changed to _cd